girder/models/collection.py

```diff
@@ -191,18 +192,74 @@ def fileList(self, doc, user=None, path='', includeMetadata=False,
                     folder, user, path, includeMetadata, subpath=True):
                 yield (filepath, file)
 
-    def subtreeCount(self, doc):
+    def subtreeCount(self, doc, includeItems=True, user=None, level=None):
         """
         Return the size of the folders within the collection.  The collection
         is counted as well.
 
         :param doc: The collection.
+        :param includeItems: Whether items should be included in the count.
+        :type includeItems: bool
+        :param user: If filtering by permission, the user to filter against.
+        :param level: If filtering by permission, the required permission level.
+        :type level: AccessLevel
         """
         count = 1
         folders = self.model('folder').find({
             'parentId': doc['_id'],
             'parentCollection': 'collection'
         })
```

Comment:
add access in fields please to match subtreeCount in folder.

    }, fields=('access',))